client/src/components/main/chatwindow/ChatWindow.js

```diff
@@ -9,12 +9,7 @@ Parent component
 State is an object that changes when comment is posted to db
 */
 
-//For testing, set jwt, user and loggedUser(in App.js)
-const jwt = '';
-
 const initialState = {
-  user: '',
-  config: { headers: { authorization: `bearer ${jwt}` } },
```

Comment:
Removed the user and config from the state because there was some lag between when a user registers and when it gets assigned, comments wouldn't render immediately and would get a 401 error until I refreshed the window